CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## 0.7.0-dev
 
+### Changed
+
+- Providing non existing paths in `import` won't result in error. Importing from these configs will be skipped
```

Comment:
I don't really like splitting sentence into 2 in changelog, and we never had something like that before. 

```suggestion
- Providing non existing paths in `import` won't result in error, instead importing configs from them will be skipped
```

@chrisduerr do you any suggestion on how to form this entry?

Reply:
We don't need a changelog entry, this was never released?

Reply:
We need, it'll go into 0.7.0, however imports will be in 0.6.0.

Reply:
Oh that's a good point. I'm not sure why I was just assuming this would be included in 0.6.0.

But I suppose it's not an issue with the existing imports system, so not having them optional should work perfectly fine.

Here would be my suggestion then:

```
 - Nonexistent config imports are skipped instead of raising an error
```

Reply:
We can cherry pick it into 0.6.0 if it's fine, since import thing is a new, so a change for it makes sense for backporting, I guess.

Reply:
Fine either way, probably easier to just get 0.6.0 out there. We'll need a follow up on imports because of home expansion anyways before it becomes **really** useful. So bundling that into 0.7.0 seems fine.

Reply:
Ok, let's move it into 0.7.0.